modules/features2d/misc/objc/gen_dict.json

```diff
@@ -1,4 +1,10 @@
 {
+    "ManualFuncs" : {
+        "SimpleBlobDetector": {
+            "setParams": { "declaration" : [""], "implementation" : [""] },
+            "getParams": { "declaration" : [""], "implementation" : [""] }
+        }
+    },
```

Comment:
Does it really work?

Reply:
Yes. I checked objc code generation and this block  just disables broken bindings.

Reply:
It'll be replaced with appropriate solution with another PR.